src/test/java/org/mockito/ArgumentMatchersTest.java

```diff
@@ -0,0 +1,69 @@
+package org.mockito;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.aryEq;
+import static org.mockito.ArgumentMatchers.cmpEq;
+import static org.mockito.ArgumentMatchers.contains;
+import static org.mockito.ArgumentMatchers.endsWith;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.find;
+import static org.mockito.ArgumentMatchers.geq;
+import static org.mockito.ArgumentMatchers.gt;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.ArgumentMatchers.isNotNull;
+import static org.mockito.ArgumentMatchers.leq;
+import static org.mockito.ArgumentMatchers.lt;
+import static org.mockito.ArgumentMatchers.startsWith;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class ArgumentMatchersTest {
+
+    public boolean actual, expected;
+
+    public ArgumentMatchersTest(boolean actual, boolean expected) {
+        this.actual = actual;
+        this.expected = expected;
+    }
+
+    @Test
+    public void verify_scenarios() {
+        assertThat(expected, is(actual));
+    }
+
+    @Parameters
+    public static List<Object[]> scenarios() {
+        return Arrays.asList(
```

Comment:
`<JustAMatterOfTaste>` 
  It should be enough to test that the methods don't return null or the expected argument matcher instance. The behaviour of the matchers are tested in there corresponding test classes, e.g: [EqualsTest ](https://github.com/mockito/mockito/blob/master/src/test/java/org/mockito/internal/matchers/EqualsTest.java) <-> [Equals](https://github.com/mockito/mockito/blob/master/src/main/java/org/mockito/internal/matchers/Equals.java).
`</JustAMatterOfTaste>`